x-pack/plugins/spaces/public/copy_saved_objects_to_space/copy_saved_objects_to_space_service.test.ts

```diff
@@ -8,12 +8,16 @@ import { ManagementSetup } from 'src/legacy/core_plugins/management/public';
 import { CopyToSpaceSavedObjectsManagementAction } from './copy_saved_objects_to_space_action';
 import { spacesManagerMock } from '../spaces_manager/mocks';
 import { CopySavedObjectsToSpaceService } from '.';
+import { notificationServiceMock } from 'src/core/public/mocks';
+import { NotificationsSetup } from 'src/core/public';
 
 describe('CopySavedObjectsToSpaceService', () => {
   describe('#setup', () => {
     it('registers the CopyToSpaceSavedObjectsManagementAction', () => {
       const deps = {
         spacesManager: spacesManagerMock.create(),
+        notificationsSetup: (notificationServiceMock.create()
```

Comment:
__question:__ wondering if `coreMock.createSetup().notifications` would be easier since it shouldn't (theoretically) require type casting.